rskj-core/src/test/java/co/rsk/core/bc/BlockValidatorTest.java

```diff
@@ -679,25 +683,59 @@ public void remascTx() {
     public void blockInTheFuture() {
         BlockGenerator blockGenerator = new BlockGenerator();
         Block genesis = blockGenerator.getGenesisBlock();
-
+        byte[] bitcoinMergedMiningHeader = new byte[0];
         int validPeriod = 6000;
+        long baseTimeStamp = System.currentTimeMillis() / 1000;
```

Comment:
I'd prefer to use deterministic values in unit tests, so results of running tests each time are same. `System.currentTimeMillis()` may cause issues

Reply:
agree, let's use a deterministic value then! I'll change it

Reply:
changed https://github.com/rsksmart/rskj/pull/1593/commits/67c63df61c8b4c075747eaaa7d01be583b4c5e4b